doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -77,8 +73,8 @@ BICR allows for the configuration of various components on your custom board, li
 * Tamper switches
 * Active shield channels
 
-You can find the details of each register contained in BICR in the relevant `BICR register's PDF file`_.
```

Comment:
With this reference gone I still think we might need to point to some other entry, otherwise the dts entries themselves can seem a bit magical.

I guess we could either link to the PS, or we can link to the dts specification for the bicr: `sdk-zephyr/dts/bindings/misc/nordic,nrf-bicr.yaml`

Reply:
I guess we will have to use the yaml file then, as the PS is still in the works.